app/assets/stylesheets/dm/components/_links.scss

```diff
@@ -1,22 +1,27 @@
 .dm-footer-link {
-  color: color($theme-color-base-ink);
-  text-decoration: none;
-  @include u-text('semibold', !important);
+  color: color($theme-color-secondary-darker);
+  @include u-text('normal');
+  text-decoration: underline !important;
```

Comment:
I think you might be able to do without this  set of `!important`s here.

Reply:
Removed!